src/peergos/shared/user/fs/FileTreeNode.java

```diff
@@ -302,6 +304,7 @@ private SigningPrivateKeyAndPublicHash getSigner() {
         return ((DirAccess) pointer.fileAccess).getChildrenLocations(pointer.filePointer.baseKey);
     }
 
+    @JsMethod
```

Comment:
what needs this in JS?

Reply:
my mistake, fixed (was used in former web-ui branch that is now redundant)